src/github-client.js

```diff
@@ -10,7 +10,7 @@ import jsonqueryEngine from 'jsonquery-engine';
 
 
 
-import Octo from 'octokat/dist/node/base';
+import Octo from 'octokat';
```

Comment:
I've been planning to reduce the size of Octokat by opting into plugins like the one that converts `snake_case` responses to `camelCase` (similar to `lodash`). I _think_ that was why I put this line in, but yep, nice catch!